smoke-tests/smoke-agent-manual.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+
+load test_helpers/utilities
+
+setup_file() {
+	echo "# 🚧" >&3
+	docker-compose up --detach collector app-agent-manual
+	wait_for_ready_app 'app-agent-manual'
```

Comment:
What a timeout on app startup looks like:

```
+++ Smoking all the tests.

cd smoke-tests && bats . --report-formatter junit --output ./
/usr/local/Cellar/bats-core/1.5.0/libexec/bats-core/bats-format-junit: line 86: name: unbound variable
/usr/local/Cellar/bats-core/1.5.0/libexec/bats-core/bats-format-junit: line 86: name: unbound variable
/usr/local/Cellar/bats-core/1.5.0/libexec/bats-core/bats-format-junit: line 86: name: unbound variable
 ✗ File ./smoke-agent-manual.bats
   🍿 Setting up app-agent-manual ... 0s ... 1s ... 2s ... 3s ... 4s ready. ✨
   (from function `wait_for_ready_app' in file ./test_helpers/utilities.bash, line 61,
    from function `setup_file' in test file ./smoke-agent-manual.bats, line 8)
     `wait_for_ready_app 'app-agent-manual'' failed
   Creating network "smoke-tests_default" with the default driver
   Creating smoke-tests_collector_1 ... 
   Creating smoke-tests_collector_1 ... done
   Creating smoke-tests_app-agent-manual_1 ... 
   Creating smoke-tests_app-agent-manual_1 ... done
 ✗ File ./smoke-agent-only.bats
   🍿 Setting up app-agent-only ... 0s ... 1s ... 2s ... 3s ... 4s ready. ✨
   (from function `wait_for_ready_app' in file ./test_helpers/utilities.bash, line 61,
    from function `setup_file' in test file ./smoke-agent-only.bats, line 8)
     `wait_for_ready_app 'app-agent-only'' failed
   smoke-tests_collector_1 is up-to-date
   Creating smoke-tests_app-agent-only_1 ... 
   Creating smoke-tests_app-agent-only_1 ... done
 ✗ File ./smoke-sdk.bats
   🍿 Setting up app-sdk ... 0s ... 1s ... 2s ... 3s ... 4s ready. ✨
   (from function `wait_for_ready_app' in file ./test_helpers/utilities.bash, line 61,
    from function `setup_file' in test file ./smoke-sdk.bats, line 8)
     `wait_for_ready_app 'app-sdk'' failed
   smoke-tests_collector_1 is up-to-date
   Creating smoke-tests_app-sdk_1 ... 
   Creating smoke-tests_app-sdk_1 ... done
   bats warning: Executed 3 instead of expected 8 tests

8 tests, 3 failures, 5 not run
```